bridge/qb/shared/main.lua

```diff
@@ -1,36 +1,43 @@
 local qbShared = require 'shared.main'
 
----@deprecated use CommaValue from imports/utils.lua
-qbShared.CommaValue = CommaValue
+---@deprecated use lib.math.groupdigits from ox_lib
+qbShared.CommaValue = lib.math.groupdigits
 
----@deprecated use RandomLetter from imports/utils.lua
-qbShared.RandomStr = RandomLetter
+---@deprecated use lib.string.random from ox_lib
+qbShared.RandomStr = function(length)
+    if length <= 0 then return '' end
+    local pattern = math.random(2) == 1 and 'a' or 'A'
+    return qbShared.RandomStr(length - 1) .. lib.string.random(pattern)
+end
 
----@deprecated use RandomNumber from imports/utils.lua
-qbShared.RandomInt = RandomNumber
+---@deprecated use lib.string.random from ox_lib
+qbShared.RandomInt = function(length)
+    if length <= 0 then return '' end
+    return qbShared.RandomInt(length - 1) .. lib.string.random('1')
+end
 
----@deprecated use string.split from imports/utils.lua
-qbShared.SplitStr = string.split
+---@deprecated use string.strsplit with CfxLua 5.4
+qbShared.SplitStr = string.strsplit
 
----@deprecated use string.trim from imports/utils.lua
-qbShared.Trim = string.trim
+---@deprecated use qbx.string.trim from modules/lib.lua
+qbShared.Trim = qbx.string.trim
```

Comment:
Once again there is a case here that [`QBShared.Trim`](https://github.com/qbcore-framework/qb-core/blob/cdd1502f0157fe2c9e1d97ef1b6e79eceb9e6c9f/shared/main.lua#L40) handles but we don't anymore.